extensions/arc/package.json

```diff
@@ -1082,7 +1085,7 @@
                               "kubeConfig": "AZDATA_NB_VAR_CONTROLLER_KUBECONFIG",
                               "clusterContext": "AZDATA_NB_VAR_CONTROLLER_KUBECTL_CONTEXT",
                               "resourceGroup": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_RESOURCE_GROUP",
-                              "connectionMode": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_CONNECTION_MODE",
+                              "connectionMode": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_CONNECTIVITY_MODE",
```

Comment:
`connectionMode` is still accurate here? Might be good to make sure the two align in naming, reduces chance of errors like this

Reply:
Actually.. this is tough because azure cli uses "connectivity-mode". I think that's where the inconsistency started. I may leave it as this for now (just changing this one piece to CONNECTIVITY_MODE because I believe that's the variable name it's supposed to have. I'll merge this one in, then open a new PR to change all instances of ConnectionMode to be ConnectivityMode instead.